DiskIOMeter.c

```diff
@@ -25,7 +25,7 @@ static const int DiskIOMeter_attributes[] = {
    METER_VALUE_IOWRITE,
 };
 
-static bool hasData = false;
+static enum { S_DATA, S_INIT, S_NODATA, S_STALE } status = S_INIT;
```

Comment:
Don't quite like this inline type declaration here. In particular due to the duplication with the other IO meter.